### ISSUE TITLE
Close ws connection if client is not subscribing to any toipic

### ISSUE THREAD

**Issue by llbartekll** [WalletConnect/WalletConnectSwiftV2#571]
> if you open a websocket and within 10seconds you’re not subscribed to anything then we close the connection with a 4008 close code

Codes: https://github.com/WalletConnect/walletconnect-docs/pull/285

**Comment by llbartekll** [comment]
@arein hey is it still relevant?

**Comment by arein** [comment]
Yup but not urgent. We currently don't send `4008` codes but one day we will start doing so.
